src/controller/c/position_sensor.c

```diff
@@ -153,6 +155,20 @@ double wb_position_sensor_get_value(WbDeviceTag tag) {
   return result;
 }
 
+double wb_position_sensor_get_velocity(WbDeviceTag tag) {
+  double result = NAN;
+  robot_mutex_lock();
+  PositionSensor *p = position_sensor_get_struct(tag);
+  if (p) {
+    if (p->sampling_period <= 0)
+      fprintf(stderr, "Error: %s() called for a disabled device! Please use: wb_position_sensor_enable().\n", __FUNCTION__);
+    result = p->velocity;
```

Comment:
```suggestion
    else
      result = p->velocity;
```

Reply:
yes, but we might need the same thing for the position, isn't it ? in wb_position_sensor_get_value
```
if (p->sampling_period <= 0)
      fprintf(stderr, "Error: %s() called for a disabled device! Please use: wb_position_sensor_enable().\n", __FUNCTION__);
    result = p->position;
```

Reply:
Yes.